Cumulus Workshop version updated and generally content edited (#83)
substrate-developer-hub/cumulus-workshop: en/3-parachains/1-launch.md

```

The first thing to notice about this command is that several arguments are passed before the lone
`--`, and several more arguments passed after it. A cumulus collator contains the actual
collator node and also an **embedded relay chain node**. The arguments before the `--` are for the
collator, and the arguments after the `--` are for the embedded relay chain node.

We give the collator a base path and ports as we did for the relay chain node previously. We also
specify the Para ID.

Remember to change the collator-specific values if you are executing these instructions a second
time for a second parachain. You will use the same relay chain chain spec, but need different ports
exposed.

> Currently, a parachain node is consisted of a parachain collator with a relay chain validator node.

Reviewer: ```suggestion
Currently, a parachain node runs both a parachain collator (or full node) tightly coupled with a relay chain full node in the same process.
There is presently no way to run a parachain node without the embedded relay chain node, but it is expected
that this will become possible for non-collator nodes eventually.
```

```

The first thing to notice about this command is that several arguments are passed before the lone
`--`, and several more arguments passed after it. A cumulus collator contains the actual
collator node and also an **embedded relay chain node**. The arguments before the `--` are for the
collator, and the arguments after the `--` are for the embedded relay chain node.

We give the collator a base path and ports as we did for the relay chain node previously. We also
specify the parachain ID.

Remember to change the collator-specific values if you are executing these instructions a second
time for a second parachain. You will use the same relay chain chain spec, but need different ports
exposed.

> Current, a parachain node is consisted of a full parachain collator with a full relay chain

Reviewer: ```suggestion
```

> validator node.

Reviewer: ```suggestion
> A Parachain node = (full) collator + (full) validator node.
> _Eventually_, this will change to only need a minimal light client for the relay chain node.
> There is also no such thing as a "light" collator node that does not include an embedded
> relay chain node _yet_ - but there will also eventually be options for this.
```
